cvs/objects/util/base/include/supply_demand_curve_saver.h

```diff
@@ -0,0 +1,56 @@
+#include <cassert>
+#include <vector>
+#include <iostream>
+#include "containers/include/imodel_feedback_calc.h"
+#include "containers/include/scenario.h"
+#include "solution/util/include/solution_info.h"
+
+using namespace std;
```

Comment:
Get rid of `using namespace` in the header file.  It doesn't appear you were actually relying on them anyhow.